File: namsor/ruby/src/ApiUsageHistoryAggregateExample.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["api_key"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::AdminApi.new.api_usage_history_aggregate

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling AdminApi#api_usage_history_aggregate: #{e}"
end

File: namsor/csharp/src/OSEG.NamsorExamples/ApiUsageHistoryAggregateExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using App.Namsor.Api;
using App.Namsor.Client;
using App.Namsor.Model;

namespace OSEG.NamsorExamples;

public class ApiUsageHistoryAggregateExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("X-API-KEY", "YOUR_API_KEY");

        try
        {
            var response = new AdminApi(config).ApiUsageHistoryAggregate();

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling AdminApi#ApiUsageHistoryAggregate: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
